tools/docker-dev/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM docker:latest as docker
+
+FROM ubuntu:latest
+ARG goversion=1.12.5
+ARG user
+
+# Install development packages.
+RUN apt-get update && apt-get -qqy upgrade && apt-get -qqy install \
+autoconf \
+autotools-dev \
+build-essential \
+curl \
+git \
+libtool \
+lsb-release \
+make \
+sudo \
+bash-completion \
+jq \
+tmux \
+vim \
+&& rm -rf /var/lib/apt/lists/*
+
+# Create user and allow sudo without password.
+RUN adduser --quiet --disabled-password --gecos ',,,,' $user \
```

Comment:
I think you want to have --uid $uid --gid $gid on that adduser line, otherwise the permission won't work when in WORKDIR.

Reply:
uid=$(id -u)
gid=$(id -g)

Reply:
Will do. I didn't need to do that to make this work on macOS. The filesharing technique is probably different.

Reply:
Fixed.